playbooks/roles/ocp-sso/tasks/main.yaml

```diff
@@ -0,0 +1,232 @@
+---
+# tasks file for playbooks/roles/ocp-sso
+
+- name: Check if cluster operators and nodes are healthy
+  include_role:
+    name: check-cluster-health
+
+- name: Setup custom catalog source
+  block:
+  - name: Include the global-secret-update role
+    include_role:
+        name: global-secret-update  
+
+  - name: Set fact variable for CatalogSource name
+    set_fact:
+      sso_catalogsource: "qe-app-registry"
```

Comment:
I think you can remove this task as the variable is already defined in the defaults/main.yaml and provide the value for "sso_catalogsource" as "qe-app-registry" instead of "redhat-operators" as defined in defaults/main.yaml.

Reply:
If validating a pre-release version for this operator the variable "sso_catalogsource" must be set as "qe-app-registry" as the openshift-test-private repo expects a catalogsource with this name or else if validating a GAed version the catalogsource as "redhat-operators" is considered

Reply:
according to the playbook logic, the value provided in default as "redhat-operators" will never be used even if you want to validate GAed version